iothub/service/src/AmqpServiceClient.cs

```diff
@@ -202,12 +196,6 @@ public override FileNotificationReceiver<FileNotification> GetFileNotificationRe
             return _fileNotificationReceiver;
         }
 
-        // This call is executed over HTTP.
-        public override Task<ServiceStatistics> GetServiceStatisticsAsync()
```

Comment:
Why are these methods being deleted? Is there a base implementation somewhere in the inheritance of this class that defines this exact implementation already?

Reply:
These methods (1) call the method (2) that takes in a Cancellation token, but by simply passing in CancellationToken.None as the token value. That is the same as calling (2) without a cancellation token, since those methods declare CancellationToken to be an optional param.